android/src/main/java/com/reactnativedidomi/DidomiModule.kt

```diff
@@ -70,6 +70,16 @@ class DidomiModule(reactContext: ReactApplicationContext) : ReactContextBaseJava
          */
         override fun noticeClickPrivacyPolicy(event: NoticeClickPrivacyPolicyEvent) = prepareEvent(EventTypes.NOTICE_CLICK_PRIVACY_POLICY.event, null)
 
+        /**
+         * The preferences is being hidden
```

Comment:
should this read something like `The preferences screen is being hidden` or `The preferences are being hidden`?

Reply:
will do the same for the notice